### PR TITLE
Removes broken documentation link for Testing Rails Applications

### PR DESCRIPTION
**Pull request by aligit**

### CODE REVIEW

**Comment by senny** [comment]
@aligit instead of removing, could we fix the link? It should probably point to `guides.rubyonrails.org`

**Comment by aligit** [comment]
@senny done :+1: 
Thanks. Do I need to submit a new pull request?

**Comment by senny** [comment]
@aligit updating the current PR is fine. However the link you put there is not pointing to the testing guide...

**Comment by aligit** [comment]
@senny It should be fine now.

File: README.md [hunk 1]
```diff
@@ -141,8 +141,8 @@ class PostPerformanceTest < ActionDispatch::PerformanceTest
 end
 ```
 
-You can find more details about the `get` and `post` methods in the
-[Testing Rails Applications](testing.html) guide.
+You can find more details about the `get` and `post` methods in the  
+[Guide to Testing Rails Applications](http://guides.rubyonrails.org/testing.html) guide.
```
Comment (senny) [inline README.md]: Why did you add "Guide to"? The sentence sounds odd.

~~Guide to~~ Testing Rails Applications **guide**.

Can you remove that again?

**Comment by senny** [comment]
thank you :yellw
